tools/rm_pyx_c_file.sh

```diff
@@ -4,3 +4,8 @@ filename="${1%.*}".c
 if [ -e "$filename" ]; then
 	rm -f "$filename";
 fi
+
+filename="${1%.*}".cpp
```

Comment:
I'd rename the script then.

Reply:
Sure, let's do that in another PR, if you want.